tests/fixtures/safe_crypto_agility.py

```
@@ -0,0 +1,13 @@
+# Safe patterns that py/hardcoded-crypto-algorithm should NOT fire on.
+import hashlib
+
+# Safe: algorithm comes from config, not a literal
+algo = config.get("hash_algorithm")
```

Ruff (0.15.10)

[error] Undefined name `config`

(F821)